docs/api-reference/geo-layers/imagery-layer.md

```diff
@@ -0,0 +1,183 @@
+import {ImageryLayerDemo} from 'website-components/doc-demos/geo-layers';
+
+<TileLayerDemo />
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/lighting-yes-blue.svg?style=flat-square" alt="lighting" />
+</p>
+
+# ImageryLayer
```

Comment:
Can we mark it as experimental, see [GlobeView](https://deck.gl/docs/api-reference/core/globe-view)

Reply:
Marked as experimental